### PR TITLE
Docs: Editing the Column Summary topic (#8535)

### PR DESCRIPTION
**Pull request by kirszenbaum**
This pull request addresses items mentioned in #8538 and #8535 (in fact, #8538 was a duplicate of one item in #8535).

I also fixed some minor grammar stuff, and rewrote a few sentences to reduce ambiguity, but I refrained form any major rewriting for now.

Re: adding an empty row to display `columnSummary` results: this seems to have already been addressed by adding the [`reversedRowCoords` option](https://handsontable.com/docs/column-summary/#setting-the-destination-cell), but I added the method described in #8538 anyway, as an alternative. Let me know if you think it makes sense. (Also, would it make sense to use the [`minSpareRows`](https://handsontable.com/docs/api/options/#minsparerows) option as well? If so, we could mention it).

Re: `forceNumeric`: to list the exact rules of how values are forced, I'd need some additional input from th DevTeam. But in the source code, I found [this](https://github.com/handsontable/handsontable/blob/master/src/plugins/columnSummary/columnSummary.js#L352-L358), which suggests that `forceNumeric` uses [parseFloat()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseFloat). I've added this info, as I think it's useful - let me know what you think.

[skip changelog]

### CODE REVIEW

File: docs/next/guides/columns/column-summary.md [hunk 1]
```diff
@@ -185,19 +201,53 @@ const hot = new Handsontable(container, {
...
+    }
+  ]
+});
+```
+:::
+
 ## Setting the calculation range
 
-By default, the plugin makes calculations on data from all rows in the endpoint's destination column. However, you can specify it differently by column and row.
+By default, the `columnSummary` plugin makes calculations on data from all rows in the endpoint's destination column. However, you can specify it differently by column and row.
```
Comment (swistach) [inline docs/next/guides/columns/column-summary.md:240]:
```suggestion
By default, the `ColumnSummary` plugin makes calculations on data from all rows in the endpoint's destination column. However, you can specify it differently by column and row.
```

**Review by swistach — CHANGES_REQUESTED** [review]
What do you think about adding some links to the API (for options, methods or a plugin)?

**Comment by kirszenbaum** [comment]
> What do you think about adding some links to the API (for options, methods or a plugin)?

@swistach - definitely. In general, I think every parameter/option/method/plugin etc. should be a link to the API ref. But I'll create a separate issue to rewrite and restructure the whole Column Summary topic (we'll agree on the content plan beforehand), and we'll add links there

**Review by swistach — APPROVED** [review]
LGTM :+1:

**Review by aninde — APPROVED** [review]
Thanks awfully!

**Pull request merged**